crates/nargo/src/cli/check_cmd.rs

```diff
@@ -63,11 +64,26 @@ fn check_from_path<P: AsRef<Path>>(p: P, allow_warnings: bool) -> Result<(), Cli
     Ok(())
 }
 
-fn build_empty_map(abi: Abi) -> BTreeMap<String, &'static str> {
-    btree_map(abi.parameters, |AbiParameter { name, typ, .. }| {
-        let default_value = if matches!(typ, AbiType::Array { .. }) { "[]" } else { "" };
-        (name, default_value)
-    })
+fn build_empty_map(
```

Comment:
Can we update this function name now? The map we're returning does not seem to be empty at all in the traditional sense. It also seems we're building something more specific than an arbitrary map.